java/sage/FSManager.java

```diff
@@ -478,7 +492,12 @@ public void releaseISOMount(java.io.File mountDir)
         }
       }
       else
-        IOUtils.exec2(new String[] { "umount", mountDir.getAbsolutePath() }, false);
+      {
+        if (Sage.LINUX_IS_ROOT)
+          IOUtils.exec2(new String[]{"umount", mountDir.getAbsolutePath()}, false);
+        else
+          IOUtils.exec2(new String[]{"sudo","umount", mountDir.getAbsolutePath()}, false);
```

Comment:
I updated sudo to use -n (non interactive) so that it will fail fast if a password is required.

```
Thu 12/1 12:26:08.888 [LibraryImporter@6b59f53b] Executing process: [sudo, -n, mount, /var/media/videos/American Sniper.iso, /tmp/formatmount, -o, loop,ro]
Thu 12/1 12:26:08.888 [LibraryImporter@6b59f53b] Started process object: java.lang.UNIXProcess@659e1620
Thu 12/1 12:26:08.890 [ErrorStreamConsumer@1408e2a5] STDERR:java.lang.UNIXProcess@659e1620: sudo: a password is required
Thu 12/1 12:26:08.891 [LibraryImporter@6b59f53b] FAILED mounting ISO image /var/media/videos/American Sniper.iso to /tmp/formatmount
```